scss/samples/_variables.scss

```diff
@@ -0,0 +1,40 @@
+// Sample Variable Documentation
+// =============================
+
+
+// Private
+// -------
+/// Private variables will display unless you set `access: ['public']`
+/// in your [SassDoc config](http://sassdoc.com/configuration/).
+/// @group samples
+/// @access private
+/// @type *
+$private: 'for internal use only';
+
+
+// Basic Value
+// -----------
+/// The basics for documenting your average non-map variable.
+/// @group samples
+/// @type *
```

Comment:
Type can be things like `string`, `map`, `color`, `length`, `number`, `list`, etc. That's all in the SassDoc annotations documents.